user_docs/en/changes.md

```diff
@@ -17,6 +17,11 @@ The available options are:
 * The timeout to perform a multiple keypress is now configurable; this may be especially useful for people with dexterity impairment. (#11929, @CyrilleB79)
 * When performing a braille cursor routing action, NVDA can now automatically speak the character at the cursor. (#8072, @LeonarddeR)
   * This option is disabled by default. You can enable "Speak character when routing cursor in text" in NVDA's braille settings.
+* Improvements in Microsoft PowerPoint text objects:
+  * It is now possible to use braille display routing keys to move the text cursor. (#9101, #17004)
+  * It is now possible to use the review cursor selection commands to select text. (#17004)
+  * Character location reporting is now accurate (e.g. when pressing `NVDA+Delete`. (#9941, @LeonarddeR)
+  * You can now move per sentence in Text frames. (#17015, @LeonarddeR)
```

Comment:
Again, I think this should be reworded to use terminology that the average user will be more familiar with. "text boxes" or "edit controls"?

Reply:
```suggestion
  * You can now move per sentence in Text boxes. (#17015, @LeonarddeR)
```